Automattic/wp-calypso
# Add the jetpack footer to the /log-in page

This pull request fixes #14983 by adding a "Powered by Jetpack" footer to the `/log-in` page.
  
<img width="459" alt="screen shot 2017-06-12 at 12 33 52" src="https://user-images.githubusercontent.com/275961/27032180-77d950f6-4f6b-11e7-9e0d-683647be23a3.png">
  
#### Testing instructions
  
1. Run `git checkout add/log-in-footer` and start your server
2. Open the [`Log In` page](http://calypso.localhost:3000/log-in)
3. Check that the "Powered by Jetpack" footer shows at the bottom of the screen
  
#### Additional notes
Test in different window sizes, to make sure it doesn't overlap content.
  
#### Reviews
